lyricsheets/models/karaoke.py

```diff
@@ -140,13 +299,62 @@ def i(self, i: int):
 
     ### End Aegisub compatibility variables
 
+    def inline_var(self, inline_name: str) -> int:
+        match inline_name:
+            case "sstart" | "start":
+                return pyasstimedelta(self._start).total_milliseconds()
+            case "send" | "end":
+                return pyasstimedelta(self._end).total_milliseconds()
+            case "smid" | "mid":
+                return pyasstimedelta(
+                    (self._start + self._end) / 2
+                ).total_milliseconds()
+            case "sdur" | "dur":
+                return pyasstimedelta(self.duration).total_milliseconds()
+            case "skdur" | "kdur":
+                return pyasstimedelta(self.duration).total_centiseconds()
+            case "syln":
+                return len(self.syls)
+            case "si" | "i":
+                return self.idxInSong
+            case "sleft" | "left":
+                return round(self.left)
+            case "scenter" | "center":
+                return round(self.center)
+            case "sright" | "right":
+                return round(self.right)
+            case "stop" | "top":
+                return round(self.top)
+            case "smiddle" | "middle":
+                return round(self.middle)
+            case "sbottom" | "bottom":
+                return round(self.bottom)
+            case "sx" | "x":
+                return round(self.x)
+            case "sy" | "y":
+                return round(self.y)
+            case "swidth" | "width":
+                return round(self.width)
+            case "sheight" | "height":
+                return round(self.height)
+            case _:
+                return 19920808
```

Comment:
thonk

Reply:
Sanity check to clearly mark an error when a non-supported inline variable is specified.